README.md

```diff
@@ -56,7 +56,13 @@ alt="Image Hub" align="left" /></a>
 <a href="https://layer5.io/projects/image-hub">Image Hub</a> is a demo application which runs on <a href="https://meshery.layer5.io/docs/service-meshes/adapters/consul">Consul</a> and facilitates exploring WebAssembly modules used as Envoy Filters.
 <br /><br /><br />
 </p>
-
+</p>
+<p style="clear:both;">
+<h2><a href="https://github.com/layer5io/meshery-operator">Meshery Operator</a></h2>
+<a href="https://meshery.io/community"><img alt="Layer5 Service Mesh Community" src="img/readme/meshery-operator-dark.svg" style="margin:10px;" width="150px" align="left" /></a>
```

Comment:
@Anita-ihuman Can you please fix the logo src here?

Reply:
whats wrong with it, please? please point the error

Reply:
There is no `img/readme/meshery-operator-dark.svg` file